src/main/java/org/isf/lab/gui/LabEditExtended.java

```diff
@@ -233,58 +233,58 @@ private JPanel getDataPanel() {
 			//exam date
 			examDateLabel = new JLabel(MessageBundle.getMessage("angal.common.date.txt"));
 			examDateLabel.setBounds(5, 10, labelWidth, 20);
-			//examDateField=getExamDateField();
-			//examDateField.setBounds(labelWidth+5, 10, 70, 20);
 			examDateFieldCal = getExamDateFieldCal();
 			examDateFieldCal.setLocale(new Locale(GeneralData.LANGUAGE));
-			examDateFieldCal.setDateFormatString("dd/MM/yyyy - HH:mm:ss");
-			examDateFieldCal.setBounds(labelWidth+5, 10, 150, 20);
+			examDateFieldCal.setDateFormatString(DATE_FORMAT_DD_MM_YY_HH_MM_SS);
+			examDateFieldCal.setBounds(labelWidth + 5, 10, 150, 20);
 			//material
 			matLabel = new JLabel(MessageBundle.getMessage("angal.lab.material"));
-			matLabel.setBounds(235, 10, 150, 20);
-			matComboBox= getMatComboBox();
+			matLabel.setBounds(255, 10, 150, 20);
+			matComboBox = getMatComboBox();
```

Comment:
here the hotfix was:
```
-			matLabel.setBounds(155, 10, 150, 20);
+			matLabel.setBounds(235, 10, 150, 20);
```
is the new size `255` meant to be?

Reply:
Fixed